packages/cli/src/commands/dev/dev.ts

```diff
@@ -124,11 +125,12 @@ export async function dev({
   const watcher = await bundler.watch(entryFile, dotMastraPath, discoveredTools);
 
   const env = await bundler.loadEnvVars();
+  const formattedEnv = convertToViteEnvVar(env, ['NO_MASTRA_TELEMETRY']);
```

Comment:
Let's change the env name
```suggestion
  const formattedEnv = convertToViteEnvVar(env, ['MASTRA_TELEMETRY_DISABLED']);
```